### PR TITLE
Updated Prerequisites for the test section

### LINKED ISSUE #502: Docker driver with new molecule version {'driver': [{'name': ['unallowed value docker']}]}
**Describe the issue**

When installing molecule docker,  with molecule version > 3.1.0 the following error takes place when trying to run a playbook test:
```
{'driver': [{'name': ['unallowed value docker']}]}
```

```
pip install molecule docker
```

**To Reproduce**

Install  molecule version > 3.1.0
```
pip install molecule docker
$ molecule converge -s plaintext-basic-rhel
..
..

CRITICAL Failed to pre-validate.

{'driver': [{'name': ['unallowed value docker']}]}
```

This is due to:
https://github.com/ansible-community/molecule/pull/2811
https://github.com/ansible-community/molecule/issues/2891

To resolve: 

```
pip3 install molecule-docker
```

### PR DESCRIPTION
**Pull request by MosheBlumbergX**
# Description

Updating pip install due to deprecation of docker driver

https://github.com/ansible-community/molecule/pull/2811

Fixes #502 

## Type of change
- [ ] updated Prerequisites, version depended

### CODE REVIEW

File: HOW_TO_TEST.md [hunk 1]
```diff
@@ -8,8 +8,15 @@ Starting with CP-Ansible 5.5.0, we have included testing via the [Molecule](http
 2. Docker installed
 3. Install the Molecule and Docker libraries
 
+
 ```pip install molecule docker```
 
+3.1 If using Molecule version > 3.1.0 install `molecule-docker`
```
Comment (JumaX) [inline HOW_TO_TEST.md:14]:
```suggestion
Note: If using Molecule version > 3.1.0 install `molecule-docker`
```

File: HOW_TO_TEST.md [hunk 2]
```diff
@@ -8,8 +8,15 @@ Starting with CP-Ansible 5.5.0, we have included testing via the [Molecule](http
...
+This is due to the removal of docker driver from [core](https://github.com/ansible-community/molecule/pull/2811).
```
Comment (JumaX) [inline HOW_TO_TEST.md:15]:
```suggestion
```

File: HOW_TO_TEST.md [hunk 3]
```diff
@@ -8,8 +8,15 @@ Starting with CP-Ansible 5.5.0, we have included testing via the [Molecule](http
...
+
+```pip3 install molecule molecule-docker```
```
Comment (JumaX) [inline HOW_TO_TEST.md:17]:
```suggestion
```

**Review by JumaX — CHANGES_REQUESTED** [review]
@MosheBlumbergX Thanks for the heads up, I've made some changes to clean up the wording.  May still want to remove some extra white space as well.  Additionally, can you rebase this to the 6.0.x branch?  We don't commit directly into -post branches for stability reasons.

**Comment by MosheBlumbergX** [comment]
Thanks @JumaX, good catch on the branch and wording.

**Review by JumaX — APPROVED** [review]
LGTM.

**Review by domenicbove — APPROVED** [review]
Looks fine!

**Pull request merged**